tests/bootstrap.php

```diff
@@ -7,6 +7,7 @@
 require_once __DIR__. '/CarrooiTests/TestCase.php';
 
 Tester\Environment::setup();
+Tester\Environment::bypassFinals();
```

Comment:
Why do we need this?

Reply:
`Mockery\Exception: The class \Nette\Application\LinkGenerator is marked final and its methods cannot be replaced. Classes marked final can be passed in to \Mockery::mock() as instantiated objects to create a partial mock, but only if the mock is not subject to type hinting checks.`